_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -134,6 +147,17 @@ class JetpackTerminationDialogFeatures extends Component {
 						</ul>
 					</div>
 				) }
+				{ connectedPlugins && (
+					<div className="jetpack-termination-dialog__generic-info">
+						<h2>{ __( 'Jetpack Connection is also used by these plugins' ) }</h2>
+						<p>
+							{ __(
+								'To fully disconnect form Jetpack you will need to disable following plugins:'
```

Comment:
Maybe we should take the number of plugins into account in our translations, and use singular or plural depending on the number of plugins we're talking about?

Here is an example:
https://github.com/Automattic/jetpack/blob/3010b47431d784ba4771188ca1da2637bb922d97/_inc/client/at-a-glance/scan.jsx#L101-L104

Reply:
I added the plural, thanks for pointing that out.